data/hai/hai reveal 4 middle.txt

```diff
@@ -1462,34 +1474,425 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	"In these proceedings, you are only a witness, here to give testimony. You will wait to speak until you are called to the witness stand. Your presence here is a privilege that can be revoked; you don't need to be in this chamber to testify."`
+			choice
+				`	"Got it. Sit until called. Speak when spoken to."`
+					goto "short explanation"
+				`	"Hours of dry proceedings aren't why I got into piloting. Can I wait this out elsewhere?"`
+					goto "please stop flooding my screen with dry text"
+				`	"I admire parliamentarian minutia and political intrigue. Now an expert stands before me! Could you explain in detail?"`
+			# The moderator's repetitive speech pattern of beginning new declarations with "Now" is intentional.
+			`	The moderator's expression changes from unemotional to impassive. "Very well, approach my station with me and I shall brief you on conduct for this session."`
+			`	You look around as you traverse the route. Although this is a formal proceeding, being observed and recorded, Elders are still filtering in and muttering to each other. Clearly, this is not yet the main event.`
+			`	There is a small desk in the chamber's central circle, opposite the entry point. It was not occupied during your report on Scar's Legion. Now, the moderator leads you to it and takes a seat, gesturing you to sit on a ledge next to it. The discomfort of your position seems to discourage lengthy arguments with the moderator.`
+			`	"Now, as a witness, you will have ample opportunity to speak when it is asked of you. You may raise your hand only to protest a matter of factual inaccuracy. You may not protest on a matter of opinion, or it will undermine your quality as a witness. Process dictates that any matter should be confirmed anyway so there should be no need.`
+			`	"Your presence," he continues quietly, "is distasteful to many Elders, and," he pauses, "presents opportunities to others."`
+			`	His dispassionate expression slips slightly and his tone reveals a hint of compassion. "Even the most hardened Hai politicians would not envy the situation you are in. You presence has great potential to both aid or harm Teeneep's case. Some may think less of you for choosing not to attend. My most solemn duty is to ensure that due representation is given to the facts. If you abide in the antechamber, your words will be relayed truthfully and sensitively. Of this you have my word.`
+			`	"You may travel between the proceedings and the antechamber at your leisure. In the antechamber is a reading room where you may find... other opportunities available. At the end of the proceedings, you may view our internal classified news network's report on the proceedings' results.`
```

Comment:
```suggestion
			`	"You may travel between the proceedings and the antechamber at your leisure. In the antechamber is a reading room where you may find... other opportunities available. At the end of the proceedings, you may view our internal classified news network's report on the results of the proceeding."`
```

Reply:
`The results of the proceeding` is still slightly better.

Reply:
Please update your suggested change, then. I'll press the "commit suggestion" button on the new suggestion.

Reply:
Done.